os/net/netdev/netdev_ioctl.c

```diff
@@ -1067,6 +1066,39 @@ int lwip_func_ioctl(FAR struct socket *sock, int cmd, void  *arg)
 
 	return ret;
 }
+
+/****************************************************************************
+ * Function: lwipioctl
+ *
+ * Description:
+ *   Call lwip_ioctl() with FIONREAD/FIONBIO commands or
+ *   call lwip API with SIOCLWIP command
+ *
+ * Parameters:
+ *   sockfd   Socket file descriptor
+ *   cmd      The ioctl command
+ *   arg      Type of the information to get
+ *
+ * Returned Value:
+ *   0 on success, negated errno on failure.
```

Comment:
Do you mean 'negative'?

Reply:
Yes.

Reply:
Yes,

Reply:
'negative' means 'minus P', 'negated' is used to express 'not P'.
When we want to get a plus/minus converted error code, the expression 'negated errno' is right.

related reference
- https://en.wikipedia.org/wiki/Negation
- https://books.google.co.kr/books?id=Ps2SH727eCIC&pg=PA45&lpg=PA45&dq=linux+negated+errno&source=bl&ots=kMGbw2ESvc&sig=ACfU3U29PwU0m0llUpIJlRGxXiz9GiA5bQ&hl=ko&sa=X&ved=2ahUKEwiEhK_v2rrgAhWXAIgKHUp2BEgQ6AEwAnoECAQQAQ#v=onepage&q=linux%20negated%20errno&f=false

Reply:
Thanks for sharing this.